Add add_data enables op_return for bdk-cli (#105)
bitcoindevkit/bdk-cli: CHANGELOG.md

- Rewrite relevant doc comments as `structopt` help document.
- Update `bdk` and `bdk-reserves` to v0.19.0.
- Change default database to `sqlite`.

> - Add the add_data option to create_tx to create an op_return output. 

Reviewer: ```suggestion
- Add `add_data` and `add_string` options in `create_tx` to create OP_RETURN outputs. 
```
